.github/workflows/devshell.yml

```diff
@@ -144,7 +144,7 @@ jobs:
         run: ./autogen.sh
 
       - name: configure
-        run: ./configure --disable-all-modules
+        run: ./configure ${DISTCHECK_CONFIGURE_FLAGS}
```

Comment:
This patch actually is needed. We discussed this with @bazsi that we should test `make distcheck` with a tarball that is built in a minimal environment.
`make distcheck` will use a different set of configure flags through `DISTCHECK_CONFIGURE_FLAGS`
He even found an issue around Python, IIRC.
We can discuss this IRL.

Reply:
Oh, I see. That is another kind if change, tho, not related to the github actions env API.

Reply:
yup. I've just found a few other cases. These might go into a separate PR though.